breathecode/certificate/tests/urls/tests_certificate.py

```diff
@@ -0,0 +1,137 @@
+            'status_code': 403,
+        })
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    @patch(GOOGLE_CLOUD_PATH['client'], apply_google_cloud_client_mock())
+    @patch(GOOGLE_CLOUD_PATH['bucket'], apply_google_cloud_bucket_mock())
+    @patch(GOOGLE_CLOUD_PATH['blob'], apply_google_cloud_blob_mock())
+    def test_delete_certificate_in_bulk_with_two_ids(self):
+        """Test / with two certificates"""
+        self.headers(academy=1)
+
+        base = self.generate_models(authenticate=True, cohort=True, user=True,
+                                    profile_academy=True, syllabus=True, capability='crud_certificate',
+                                    cohort_user=True, specialty=True, role="potato"
+                                    )
+        del base['user']
+
+        model1 = self.generate_models(user=True, profile_academy=True, user_specialty=True,
+                                      user_specialty_kwargs={'token': "hitman3000"}, models=base)
+
+        model2 = self.generate_models(user=True, profile_academy=True, user_specialty=True,
+                                      user_specialty_kwargs={'token': "batman2000"}, models=base)
+
+        url = reverse_lazy('certificate:certificate_academy') + '?id=1,2'
+        response = self.client.delete(url)
+
+        if response.status_code != 204:
+            print(response.json())
+
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+        self.assertEqual(self.all_user_invite_dict(), [])
+
+    @patch(GOOGLE_CLOUD_PATH['client'], apply_google_cloud_client_mock())
+    @patch(GOOGLE_CLOUD_PATH['bucket'], apply_google_cloud_bucket_mock())
+    @patch(GOOGLE_CLOUD_PATH['blob'], apply_google_cloud_blob_mock())
+    def test_delete_certificate_in_bulk_not_found(self):
+        """Test / with two certificates"""
+        self.headers(academy=1)
+
+        base = self.generate_models(authenticate=True, cohort=True, user=True,
+                                    profile_academy=True, syllabus=True, capability='crud_certificate',
+                                    cohort_user=True, specialty=True, role="potato"
+                                    )
+        del base['user']
+
+        model1 = self.generate_models(user=True, profile_academy=True, user_specialty=True,
+                                      user_specialty_kwargs={'token': "hitman3000"}, models=base)
+
+        model2 = self.generate_models(user=True, profile_academy=True, user_specialty=True,
+                                      user_specialty_kwargs={'token': "batman2000"}, models=base)
+
+        url = reverse_lazy('certificate:certificate_academy') + '?id=3,4'
+        response = self.client.delete(url)
+
+        if response.status_code != 204:
+            print(response.json())
+
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
```

Comment:
It should be 404 i think

Reply:
If it is a normal varidation error with status 400, you should validate the slug with `response.json()`